src/component/DigitButton/index.jsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styles from "./index.module.css";
+
+function DigitButton({ value, onClick }) {
+  return (
+    <button className={styles.digit} onClick={() => onClick(value)}>
+      {value}
+    </button>
+  );
+}
+
+DigitButton.propTypes = {
+  value: PropTypes.number.isRequired,
+};
```

Comment:
```suggestion
function DigitButton({ value, onClick }) {
  return (
    <button className={styles.digit} onClick={() => onClick(value)}>
      {value}
    </button>
  );
}

DigitButton.propTypes = {
  value: PropTypes.number.isRequired,
  onClick: PropTypes.func.isRequired
};
```

function도 추가가 가능하네요!
react/prop-types 린트에 걸려서 추가해두는게 좋아보여요 👀

Reply:
감사합니다~ 제안해 주시는 부분은 제가 코드 하나씩 살펴보면서 수동으로 반영해 둘게요!